src/components/JobLogs/JobLogs.js

```diff
@@ -86,6 +86,7 @@ class JobLogs extends Component {
         formatter={formatter}
         sortOrder={this.columnMapping[sortField]}
         sortDirection={sortDirection}
+        autosize
```

Comment:
Why do I see three commits in the history whereas I only expect one? Please check this.